pyo3-macros-backend/src/quotes.rs

```diff
@@ -1,23 +1,23 @@
 use crate::utils::Ctx;
-use proc_macro2::TokenStream;
-use quote::quote;
+use proc_macro2::{Span, TokenStream};
+use quote::{quote, quote_spanned};
 
 pub(crate) fn some_wrap(obj: TokenStream, ctx: &Ctx) -> TokenStream {
-    let Ctx { pyo3_path } = ctx;
+    let Ctx { pyo3_path, .. } = ctx;
     quote! {
         #pyo3_path::impl_::wrap::SomeWrap::wrap(#obj)
     }
 }
 
-pub(crate) fn ok_wrap(obj: TokenStream, ctx: &Ctx) -> TokenStream {
-    let Ctx { pyo3_path } = ctx;
-    quote! {
+pub(crate) fn ok_wrap(obj: TokenStream, ctx: &Ctx, span: Span) -> TokenStream {
```

Comment:
If I saw it correctly this is also the `output_span`, You could extract that directly from the `ctx` instead of passing it again, right?

Reply:
I originally did it this way because I wasn't sure the span was always correct (i.e., whether ok_wrap is exclusively used on return values) but I think it is, so I've removed it now.